gui/src/main.rs

```diff
@@ -215,8 +215,23 @@ impl Gui {
             .into_iter()
             .map(|r| (r.id.clone(), r))
             .collect();
-        let tabs_by_id: HashMap<String, Tab> =
-            saved_tabs.into_iter().map(|r| (r.id.clone(), r)).collect();
+        let tabs_by_id: HashMap<String, Tab> = if saved_tabs.len() > 0 {
```

Comment:
Ran into an error trying to start postie if you accidentally closed it w/o any tabs open